janggi/gameboard.py

```diff
@@ -127,6 +127,13 @@ def set_turn(self, color):
         """setter for turn"""
         self._turn = color
 
+    def get_next_turn(self):
+        """getter returns the next player's turn"""
+        if self._turn == "b":
+            return "r"
+        else:
+            return "b"
+
```

Comment:
Side Note: you could do this in a ternary operator sort of way...

```
return self._turn == "b" and "r" or "b"
```

Actual Comment: we have this "flip-the-color" logic in a few places. Might be worth adding a helper function that simply does that. Which would then give you a definition of...

```suggestion
        def swap_color(color):
            return 'b' == color and 'r' or 'b'
    
        def get_next_turn(self):
            return swap_color(self._turn)
```

That `swap_color()` function could then be used wherever else we toggle colors (and it would eventually belong in my forthcoming `utils.py` module).

Reply:
That's a good idea! I've never used a ternary operator like that

Reply:
Is that return statement kinda saying "if 'b' == color return 'r', else return 'b'"? cause that's incredible...

Reply:
Yeah, exactly.